### DISCUSSION TITLE
Active Threads Per Thread Group

### DISCUSSION THREAD

**Discussion by shaqb** [ludeknovy/jtl-reporter#220] [Ideas]
Hey @ludeknovy - another one from me if possible!

We had a situation in our load test where we had threads dropping out. We can see that in the graph here:
![image](https://github.com/ludeknovy/jtl-reporter/assets/19668009/2aa81aab-02fd-4afa-b7fd-25623aa8d9d8)

However we have many thread groups in our scenario and it's hard to see which thread group was dropping. When we use the Jmeter report we can filter by thread group like this:
![image](https://github.com/ludeknovy/jtl-reporter/assets/19668009/9b594d48-6001-47dd-a88d-619298ac174c), and it's clear which one has the issue.

Would it be possible to have something similar? We would then be able to determine which thread group had the issue and target that one accordingly.

**Comment by ludeknovy** [comment]
Hi @shaqb , yes, this should be possible as well.

**Reply by ludeknovy** [reply]
@shaqb Would you mind contacting me at ludek@jtlreporter.site ?

**Comment by shaqb** [comment]
Hi @ludeknovy long time no speak! I was wondering if you ever got a chance to implement this feature?

**Reply by ludeknovy** [reply]
Hi @shaqb I remember looking into it, but not sure where I got. Gonna check it again soon. Sorry.

**Reply by ludeknovy** [reply]
Ahh, actually it looks like I’ve finished it, see https://github.com/ludeknovy/jtl-reporter/issues/221

**Comment by shaqb** [comment]
Nice! Thanks dude!
